extension/resources/walkthrough/more-resources.md

```diff
@@ -30,7 +30,7 @@
 [python api]: https://dvc.org/doc/api-reference
 [free course]: https://learn.iterative.ai/
 [contribute]: https://dvc.org/doc/contributing/core
-[example project]: https://github.com/iterative/example-get-started
+[example project]: https://github.com/iterative/example-dvc-experiments
```

Comment:
I believe we want to direct users to `example-dvc-experiments` instead of `example-get-started`

Reply:
I am not sure if this is correct or if it should now be https://github.com/iterative/example-get-started-experiments. Probably best to ask @shcheklein / @dberenbaum.

Reply:
Fine to make this change and follow up

Reply:
and there are multiple instances of this repo being linked to.

Reply:
yes, it should be https://github.com/iterative/example-get-started-experiments

Reply:
Links have been updated!